core/sms_server_v2.py

```
@@ -0,0 +1,592 @@
+    except Exception as e:
+        logger.error(f"Error stopping workers: {e}")
+    
+    # Backup Redis state to Postgres
+    try:
+        from core.database import get_db_context
+        redis_client.backup_to_power_down_store(
+            get_db_context().__enter__(),
+            None
+        )
+        logger.info("Redis state backed up")
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)